nucypher/blockchain/eth/actors.py

```diff
@@ -440,27 +407,74 @@ def _is_phase_1_action_required(self, ritual_id: int) -> bool:
 
         return True
 
+    @staticmethod
+    def _phase_id(ritual_id: int, phase: int) -> Tuple[RitualId, PhaseId]:
+        return RitualId(ritual_id), PhaseId(phase)
```

Comment:
It's strictly for type annotations (there many `int`s flying around and in particular I'm helping devs not accidentally get them swapped in future iterations).  Internally I tried to use a this type of tuple everywhere possible and this is it's "constructor".  I suppose it can be promoted to a `NamedTuple` and it will clean this up even more.

Reply:
> I suppose it can be promoted to a NamedTuple and it will clean this up even more.

That's what I was thinking. That being said, I trust your judgement, so your call.

Reply:
✔️  Good call - it's way more formal as a `NamedTuple` (`PhaseId` can be shorthanded as `pid`)